src/lib/containers/table/TopLevelControls.tsx

```diff
@@ -119,6 +120,28 @@ const TopLevelControls = (props: TopLevelControlsProps) => {
   const refresh = () => {
     executeQueryRequest(getLastQueryRequest())
   }
+
+  /**
+   * We show the total number of results that would be shown if the user removed their filters.
+   * To do this, we have to create a query that captures those results.
+   */
+  const unfilteredResultsQuery: Query = useMemo(() => {
+    const initQueryRequest = getInitQueryRequest()
+    return {
+      sql: initQueryRequest.query.sql,
+      selectedFacets: (initQueryRequest.query.selectedFacets ?? []).filter(
+        facet => facet.columnName === lockedColumn?.columnName,
+      ),
+      additionalFilters: (
+        initQueryRequest.query.additionalFilters ?? []
+      ).filter(qf =>
+        'columnName' in qf
+          ? qf['columnName'] === lockedColumn?.columnName
+          : true,
+      ),
+    }
+  }, [getInitQueryRequest, lockedColumn?.columnName])
+
```

Comment:
Need to include selectedFacets/additionalFilters now since those may be applied within a details page